.buildkite/pipeline.yml

```
@@ -34,28 +33,25 @@ steps:
     if: &not_scheduled
       build.source != "schedule"
     command:
-      nix run -f. pkgs.hlint -c ./scripts/lint.sh
+      nix shell .#haskellPackages.hlint -c ./scripts/lint.sh
 
-  - label: reuse lint
-    command: nix run -f. pkgs.reuse -c reuse lint
-    if: *not_scheduled
-
-  - label: check trailing whitespace
-    command: .buildkite/check-trailing-whitespace.sh
+  - label: check
     if: *not_scheduled
+    command:
+      nix flake check -L
 
   - label: xrefcheck
-    command: "nix shell -f . pkgs.xrefcheck -c xrefcheck"
+    command: "nix shell .#xrefcheck --impure -c xrefcheck"
```

`xrefcheck` was flakified, so now you can do something like:
```suggestion
    command: nix run "github:serokell/xrefcheck?rev=d1f465048eb2d9f345dbcfcc645a07440c7ebe80"
```
After that you can safely remove all mentions of `xrefcheck` from `flake.nix`